README.md

```diff
@@ -1 +1,44 @@
-# Visualizing-Word-Embeddings-using-CBOW-Model
\ No newline at end of file
+# Visualizing-Word-Embeddings-using-CBOW-Model
+Visualizing word embeddings by implementing the CBOW (continious bag of words) model and using shallow neural network with one fully connected dense hidden layer
+
+**Dataset**
+- [The Complete Works of William Shakespeare](https://www.kaggle.com/kewagbln/shakespeareonline)
+
+**Classes**
+- DatasetProcessing : class to process dataset, get the required parameters and print info on interface using standard logger
+- Model : class to define the shallow neural network model, gradient descent and word embeddings
+- PlotCode : class to generate and save 2D and 3D plots of the word embeddings from the trained model
+
+**Utility**
+- utilities : bundle of helper functions for global use in this project
+
+**Executables**
+- TrainModel : Initializes and trains the neural network, extracts word embeddings and generates 2D and 3D plots to visualize the word embeddings
+
+**Results**
+- 2D Plots of word embeddings : <br />
+    word embeddings after 20 iterations <br />
+    <img src="Images/2D/2D_word_embeddings_num_iters_20.png" width=1000> <br /><br />
+    word embeddings after 40 iterations <br />
+    <img src="Images/2D/2D_word_embeddings_num_iters_40.png" width=1000> <br /><br />
+    word embeddings after 60 iterations <br />
+    <img src="Images/2D/2D_word_embeddings_num_iters_60.png" width=1000> <br /><br />
+    word embeddings after 80 iterations <br />
```

Comment:
Not clear what the embeddings are in the graph or what improvement each successive iteration is bringing